### ISSUE TITLE
msg.Change value

### ISSUE THREAD

**Issue by LK4D4** [vishvananda/netlink#38]
0207b6b6339670b6f4e9ddd2c2e72ce8d567969e changed behavior a little.
From `man 7 rtnetlink`:
```
ifi_change is reserved for future use and should be always set to 0xFFFFFFFF.
```
Also when set to `0xFFFFFFFF` any netlink call sets interface up(have no idea why). So, I'm not sure what sense make set `msg.Change` to anything. It works with any value, but with `0xFFFFFFFF` it also sets interface up :)
ping @vishvananda @marek-polewski

**Comment by LK4D4** [comment]
I looked in libnl code and `ifi_change` isn't used there at all.

**Comment by vishvananda** [comment]
That is odd. I'm curious about the issue with it overwriting flags on SetLink though that @marek-polewski referenced in the pull request

**Comment by LK4D4** [comment]
@vishvananda Odd indeed, and I'm not sure what expected behavior.

**Comment by vishvananda** [comment]
Fixed by #44
